src/tests/tracing/eventpipe/processinfo3/processinfo3.cs

```diff
@@ -233,10 +233,11 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.RiscV64 => "riscv64",
```

Comment:
Please also add loongarch64 while updating these.

Reply:
I've added loongarch64 too, but please note that I've not verified that test passes on it now. @shushanhf @LuckyXu-HF please take a look.

Reply:
Thanks. Yes, it's funny that either that test is suppressed or it throws anyway. We have these unnecessary platform maps in too many places which add no value (just another string to update for the next platform).

Reply:
Thank you for heads up. I have verified on LA64 and these are also suitable for LA64.
LGTM, thanks very much.